### PR TITLE
Add formatting

### PR DESCRIPTION
**Pull request by szabo137**

### CODE REVIEW

**Comment by szabo137** [comment]
TD;DR

This PR will *not* trigger the formatter job as discussed here: https://github.com/QEDjl-project/QEDbase.jl/pull/19
Nevertheless, since this is copied from the other packages, it can be merged without checking the trigger (because it worked after merging in the other repos as well).

File: .formatting/Manifest.toml [hunk 1]
```diff
@@ -0,0 +1,226 @@
+# This file is machine-generated - editing it directly is not advised
```
Comment (SimeonEhrig) [inline .formatting/Manifest.toml:1]: Is it wanted, that his file is in the PR?

Comment (szabo137) [reply .formatting/Manifest.toml]: Removed in 8321d60 .

**Comment by szabo137** [comment]
@SimeonEhrig I think this is ready for merging 🙆‍♂️

**Review by AntonReinhard — APPROVED** [review]

**Pull request merged**